SPECS-EXTENDED/frr/frr.spec

```diff
@@ -95,7 +93,12 @@ SELinux policy modules for FRR package
 %endif
 
 %prep
-%autosetup -p1 -n %{name}-%{name}-%{version}
+%setup -n %{name}-%{name}-%{version}
+%patch0 -p1
+%patch1 -p1
+%patch2 -p1
+%patch3 -p1
+%patch4 -p1
```

Comment:
This change can be reverted. `%autosetup` applies these patches for us, and removes the possibility that we might forget to apply patches manually.